conf/conf.go

```diff
@@ -0,0 +1,35 @@
+package conf
+
+import (
+	"encoding/json"
+	"os"
+)
+
+// Configuration loads the config file
+type Configuration struct {
+	RootDirectory string
+	IndexFilename string
+	Port          string
+}
+
+// NewConfig is a function for creating a new configuration
+func NewConfig(filename string) Configuration {
+	config := Configuration{}
+	name := "config.json"
+	if filename != "NULL" {
+		name = filename
+	}
+	file, err := os.Open(name)
+	defer file.Close()
+	must(err)
+	decoder := json.NewDecoder(file)
+	err = decoder.Decode(&config)
+	must(err)
+	return config
+}
+
+func must(e error) {
```

Comment:
This is repeated in every package can you make the change in one and use it everywhere?

Reply:
I was thinking was creating a separate logger module which could take care of this thing, something like this [one](https://github.com/hellozee/cook/blob/master/logger/logger.go)

Reply:
makes sense lets do this

Reply:
let's do this

Reply:
Shouldn't we prepare a separate patch for this?